eo.json (Jackalgirl)

Jackalgirl opened the pull request:
See: https://github.com/Yellow-Dog-Man/Locale/pull/65#issuecomment-1753976671

For collaborative worklist

RobertBaruch commented:
I think "Dummy" might be required to remain "Dummy"?

**Asset**

Asset is probably _aktivaĵo_. "Asset" is really a finance term, and I think it was appropriated by marketers for "digital assets" (e.g. Unity Asset Store), and then probably by game developers to describe, well, pretty much anything.

_Havaĵo_, though, just means "a belonging", something that belongs to someone.

Komputeka claims that asset is _pluso_, but I think that means more like, something good.

**360-foto**

Change to _foto de 360°_ as you did for the others.

**Skybox**

I might use _cielkubo_ instead, since _kesto_ (box) is closer to a chest, emphasizing the container -- something you put things in. A skybox is just a cube with textures on the inside :>

**LayoutVerticalLR et al**

Not really a fan. I might just stick with _horizontala/verticala_ and _maldextren/dextren_. These are just telling you the layout direction (e.g. horizontal, left to right).

**AutoScale**

_Aŭtomate skali_ (I think this command just means, to automatically scale the imported model)

**Normals**

Normal = _ortanto_. Although _normalo_ is fine, _ortanto_ is a synonym, not confusable with "normal" in its other senses, and is also closer to the form of _tanĝanto_.

**VertexColor, AlbedoColor**

I am annoyed at English adjectives. English uses words to modify other words in a more vague way, so _blue box_ means _a box that is blue_, but _cat box_ is _a box for cats_. In both cases, one word modifies the other. However, in Esperanto, if you said _kata kesto_, it would mean _a box that is a cat_, or _a box that has the characteristic of a cat_. So you'd have to say _kesto por katoj_. This holds for most words where the noun is the root word.

In cases where a noun is used to modify another noun, Esperanto **compounds** them, so instead of _kesto por katoj_ you could just say _kato-kesto_ or _kat-kesto_ or _katkesto_. Another example, cardboard box is not _kartona skatolo_, but _kartonskatolo_. You could also say _skatolo de kartono_ (_a box made of cardboard_).

So in this case, vertex color should be _vertic-koloro_, and albedo color should be _albed-koloro_.

Try not to compound more than two words. Use prepositions if you need more.

(end of part 1)

RobertBaruch commented:
TextureAlpha -> Tekstur-alfo (alfo de teksturo)

Emissive: I think your analysis is good. I'd use emisianto/emisianta, "something that emits" rather than emisema, "prone or included to emit". Emissives only do emission, not anything else.

Deduplicate Instances: I've used _malmultobli_ (the exact opposite of making multiple copies).

Mesh: In general, I use maŝaro, since maŝo is just one small element of a mesh, like a loop.

Flatshaded: you used nuanco, which is a different sense of shade (a small difference). Maybe kolorheleco (color lightness) or ombrigita (shaded)

Devigi + -n: I think the -n ending is for the thing being forced, e.g. devigi min esti bona (to force me to be good). "Force T-Pose" is shorthand for "force (something to be) T-pose", so maybe devigi T-poso?

Orb: Globo is nice.

Headset: Great analysis, I think surkaska interfaco is almost right -- surkaska kamerao would be a camera _on_ a helmet. Perhaps in keeping with the noun-noun modification, surkask-interfaco?

Reset: Restartigi is the right term!

Calibration: Well, at least in English, that is the etymology: to measure the caliber of something. The meaning got extended. Most of the [languages I checked](https://en.wiktionary.org/wiki/calibrate#Translations) are fine with using kalibri, which probably means that if you say kalibri in Esperanto, nobody will think it means "to measure the caliber".

Jackalgirl commented:
**Adjectives & Compounds**
You are absolutely right, I had completely forgotten about that.  Thank you!  I will fix. : )

**LayoutVerticalLR et al**
Understood; this was actually in the original. So I had to go into Resonite and figure it out from context.  It's for stereoscopic images, and they can either be side-by-side or top over bottom, left (or right) (eye), and right (or left) eye.  So I changed it to "Supra / Suba (Maldekstra / Dekstra)" etc.  (assuming that the word "image" is left out, as in "Top image, bottom image" and so forth).  I figure that if you're in the weeds enough to be importing stereoscopic images, this would make sense, but I'm still waffling back and forth whether to make "Supra" et al a noun, like "Flanko ĉe Flanko", though now I'm wondering if "unu apud la alia" is better...

**AutoScale**
I completely agree -- however in this case the appearance of these words is in response to a question.  Q: "In what kind of units/measurement is this thing?" A: "(It's in an) automatic scale (unknown scale)".

**Asset**
Well, this begs a deeper, more profound question.  Which is it?  Is it "asset" in a financial sense, or are these things that belong to someone (or something)?  I'm not being snarky; I think there's an important philosophical underpinning here.  In English, it could have either meaning.  But in Esperanto, we can be much more specific: if we think that these things are elements that either belong to a player, a game, or another object (mesh, slot, whatever), maybe "havaĵo" is a better word?

**Ĉielkubo/kesto**
The reason why I'd used "kesto" for box was because in Komputeko, "kesto" is reference for "bounding box", and I was thinking of a skybox as a bounding box, but I see your point too, and agree -- thank you!

**Kalibri**
Thanks for your thoughts on "kalibri"; I still feel like I"d like to leave it as "kalibrigi", though, simply because per the PIV, "kalibrigi" means "Fari, ke iu korpo havu ian precizan diametron", which is a lot closer to what we're doing here.  That's why I was moving in that direction; to retain the original meaning of "kalibri" per the PIV, and then reorient everything around "kalibrigi" since the definition is closer.  This resolves the issue of "kalibrilo" being "caliper" (which is not an archaic term; I have a caliper right next to me on my desk as I type) -- while a caliper is a calibration tool, I don't think this calibration tool in Resonite is a caliper (squares == rectangles != squares).

**Surkaska Interfaco**
Oh, I absolutely love "VR-Surkasko" -- this is so much better, thank you!  I feel like it's just more obvious and less technical: "VR Helmet" rather than "Helmet Interface".  GLEE!

**Shade**
This is a tough one. On the one hand, we have "shade" as in "to apply shadows", or "a place characterized by a shadowy quality".  That is the "ombr-" root.  On the other hand, we have "shade" as a reference to the value (light/darkness) of a color (and often conflated with saturation, the intensity of its color) (or, as the PIV would have it, "[d]elikata, apenaŭ sentebla diferenco inter gradoj k kombinoj de iu koloro".  That is "nuanc-".  I think, though, that I am leaning towards "ombr-" for this as you've said, because I think that at the end of the day, when we're talking about rendering and shaders, we're talking about how an object's surface is depicted in the context of light (or its absence), and applies to making the lit parts look lit, even though we're using the word "shade" in doing so (which makes me unhappy).

**Devigi+n**
Devigi+n the gremlin in me wants to say it means "Devigi en T-Pozon" (which suggests, to me, accusative because it's flavor of "motion towards" -- "force in*to* T-Pose).  What do you think? ; ) (I wink, but I'm also kind of serious.  The "NE FORGESU LA AKUSATIVON" creature is always sitting on shoulder ;) )

**Everything Else**
In accordance (I grok *maŝaron*), changes made, thank you!

P.S. I'm *supposed* to be working on other work right now.  Could you all be a little less interesting, please? ; )

P.P.S. Kiam ni ĉiuj kunvenos Esperante en Resonite?

Jackalgirl commented:
I haven't changed "Dummy" yet, though.  Double-checking. : )

RobertBaruch commented:
For **side-by-side**, _flank-al-flanke_ would probably be best.

Hmm, for **asset** I'm still not sure whether it's aktivaĵo or havaĵo. Checking on other languages for the computer sense, Spanish, French and Ukrainian seem to use some form of _resource_ (Esperanto _risurco_), while some other languages use some form of _possession_ (_havaĵo_). Maybe _risurco_?

With _nuanc-_, I sometimes forget to check ALL the sources, otherwise I'd have found the PIV definition :> Most languages give up and use "shader", but Spanish and Portuguese in particular use the equivalent of _ombrigilo_. I know, that would imply only applying shadows, but maybe sometimes we have to bow to other languages?

I like "Devigi en T-Pozon"!

Mia nomo sur Resonite estas Xekri. Bonvolu sendi al mi kontaktpeton!

Jackalgirl commented:
Okay, re the "Dummy" line -- the instructions say:

"Always keep the "Dummy" : "Dummy" entry at the bottom of the file. This way you don't have to remember to remove the comma at the end of the last entry every time"

I am reading this not as "Dummy" has to say "Dummy" but "the last entry of a list like this in json shouldn't have a comma after it, so don't copy in or add any entries after this line; leave it at the bottom, as it doesn't have a comma and is therefore (and should remain) the Last Line of the list."

Funny, though, all of the language entries have left the translation of "Dummy" as "Dummy".  I'm going to have to ask in PrimeTime as this makes no sense to me. ; )

RobertBaruch commented:
Right, okay. Maybe just leave it untranslated, since it's quoted in the instructions that way.

Jackalgirl commented:
I will revert it and pend the chance until I've had a chance to ask in PrimeTime; I think it's one of those things that is vanishingly unlikely to ever be seen or used, but still. ; )

Jackalgirl commented:
Finished several new sections.  Also updated the "author" section to add the three of us.  If this has been done, I can fix.  My focus for the time being is the sections that are new to Resonite, so that the locale file isn't missing anything.

Jackalgirl commented:
Of note: Prime says: "Dummy" isn't actually used anywhere in the codebase; it exists solely as an entry for the benefit of translators (to mark the end of the list) and to let it be.  So I am letting it be. : )

Jackalgirl commented:
Moving the "Work List" over here for the time being.

Drafted | Reviewed | Approved for Merge/Pull-Request Submitted

- [x] :ballot_box_with_check: :black_square_button: General.* - XDelta
- [x] :ballot_box_with_check: :black_square_button: **Worlds.\* (new)** - XDelta
- [x] :ballot_box_with_check: :black_square_button: Locomotion.* - XDelta
- [x] :ballot_box_with_check: :black_square_button: Interaction.* - XDelta
- [x] :ballot_box_with_check: :black_square_button: **Inventory.\* (new)** - XDelta
- [x] :ballot_box_with_check: :black_square_button: Dash.* - XDelta
- [ ] :black_square_button: :black_square_button: **Facets.\* (new)** - Jackalgirl
- [x] :ballot_box_with_check: :black_square_button: World.* - XDelta
- [ ] :black_square_button: :black_square_button: Tools.* (Note: see towards bottom of list for separate section Tools.Names.*)
- [ ] :black_square_button: :black_square_button: Options.*
- [x] :ballot_box_with_check: :black_square_button: Indicator.* - XDelta
- [x] :black_square_button: :black_square_button: **Login.\* (new)** - Jackalgirl - #65 
- [x] :black_square_button: :black_square_button: **Register.\* (new, and split up within Login.\* section)** - Jackalgirl - #65 
- [ ] :black_square_button: :black_square_button: Account.*
- [x] :black_square_button: :black_square_button: **Avatar.\* (new)** - Jackalgirl - #65 
- [ ] :black_square_button: :black_square_button: 2FA.*
- [ ] :black_square_button: :black_square_button: Profile.*
- [x] :ballot_box_with_check: :black_square_button: Session.* - XDelta
- [ ] :black_square_button: :black_square_button: Permissions.*
- [ ] :black_square_button: :black_square_button: User.*
- [ ] :black_square_button: :black_square_button: Contacts.*
- [ ] :black_square_button: :black_square_button: Notifications.*
- [ ] :black_square_button: :black_square_button: Undo.*
- [ ] :black_square_button: :black_square_button: Tooltips.*
- [ ] :black_square_button: :black_square_button: Tooltip.*
- [ ] :black_square_button: :black_square_button: CreateNew.*
- [ ] :black_square_button: :black_square_button: AvatarCreator.*
- [x] :black_square_button: :black_square_button: Importer.* - Jackalgirl - #65 
- [x] :black_square_button: :black_square_button: Exporter.* - Jackalgirl - #65 
- [x] :black_square_button: :black_square_button: NewWorld.* - Jackalgirl - #65 
- [x] :black_square_button: :black_square_button: FullBody.* - Jackalgirl - #65 
- [ ] :black_square_button: :black_square_button: CameraControl.*
- [ ] :black_square_button: :black_square_button: Settings.*
- [ ] :black_square_button: :black_square_button: Security.*
- [x] :black_square_button: :black_square_button: **ComponentSelector.\* (new)** - Jackalgirl - #65 
- [ ] :black_square_button: :black_square_button: Inspector.*
- [x] :black_square_button: :black_square_button: Wizard.* - RobertBaruch - #57 
- [ ] :black_square_button: :black_square_button: Desktop.*
- [x] :black_square_button: :black_square_button: **VolumePlaneSplicer.\* (new)** - Jackalgirl
- [ ] :black_square_button: :black_square_button: Tutorial.* (note: two items now in a section following "CloudHome", and more under "StickyNote")
- [ ] :black_square_button: :black_square_button: Object.*
- [ ] :black_square_button: :black_square_button: Twitch.*
- [ ] :black_square_button: :black_square_button: Exiting.*
- [x] :black_square_button: :black_square_button: **Migration.\* (new)** - Jackalgirl
- [ ] :black_square_button: :black_square_button: **Temporary.\* (new)** - Jackalgirl
- [ ] :black_square_button: :black_square_button: **CloudHome.\* (new)** (Note: some items follow StickyNote) - Jackalgirl
- [ ] :black_square_button: :black_square_button: **Mirror.\* (new)**
- [ ] :black_square_button: :black_square_button: **StickyNote.\* (new)**
- [ ] :black_square_button: :black_square_button: **Hints.\* (new)**
- [x] :ballot_box_with_check: :black_square_button: **UI.\* (new)** - XDelta
- [x] :black_square_button: :black_square_button: **Tools.Names.\* (new)** - RobertBaruch - #57 
- [x] :black_square_button: :black_square_button: Dummy - Jackalgirl - #65 

Note: Fixed explicit assignments -- let me know if there are sections you are working on, or if there's a sub-section you want to take on (if you don't want to do an entire section), let me know and I can update. : )

Jackalgirl commented:
Okay.  I think I am caught up with all of the Resonite-new sections.  I will wait until the previous pulls are merged, then go through and make sure that my file is updated with all of those changes, before undrafting this one. : )

I do still have to proofread and check for dropped -n's and all that jazz.  Commentary greatly appreciated. : )

Jackalgirl commented:
Pulling this out for visibility:

I've verified this file line-by-line against en.json.  This required the importing and translation of a cubic butt-ton of items; a solid proofread by the Esperanto community would be greatly appreciated.

Also, note: the typo in line 2007 (XDelta's issue https://github.com/Yellow-Dog-Man/Locale/issues/12), "UI.VideoPlayer.BroascastAudio", is present in this file.

XDelta commented:
> Pulling this out for visibility:
> 
> I've verified this file line-by-line against en.json. This required the importing and translation of a cubic butt-ton of items; a solid proofread by the Esperanto community would be greatly appreciated (I'm very tired).
> 
> I'd like to add that I've chosen a new word, "Esprimocelo", for "Blendshape", based on the fact that the whole point of blendshapes is to set up an array of states in a mesh's shape that represent expressions, and then drive expressions by weighting those expressions-as-shapes. I didn't feel like a direct, one-to-one translation worked here; since the shapes are targets that represent the ideal 100% (expression) expression, I thought "Esprimoceloj" (expression targets) would be more appropriate. If there are any Esperantists who work in 3D animation in Esperanto and use a different word for this, I will of course totally defer to you. : )
> 
> Also, note: the typo in line 2007 (XDelta's issue #12), "UI.VideoPlayer.BroascastAudio", is present in this file.

The typo in a locale key is fine, we don't need to correct the keys. They'll be automatically fixed later as mentioned in the readme

Jackalgirl commented:
I'm going to tentatively leave this as draft until the end of the weekend (whenever I go to bed on Sunday, Oct 22), unless I forget, which is entirely possible.  Please add any comments if you see anything glaring or would suggest a different translation.  If you need more time than that, no worries; let me know and I'll delay.  Dankegon!

behilam commented on eo.json:
```diff
@@ -305,24 +371,135 @@
+        "Register.MalformedUID": "Nevalida aparatinformo",
+        "Register.MalformedUID.Description": "Bonvolu provi reinstali, aŭ kontakti subtenadon je support.resonite.com.",
+
+        "Login.MissingLogin": "Neniuj indentigiloj provizataj",
+        "Login.MissingLogin.Description": "Ĉu vi forgesis provizi itentigilojn?",
```
Oni provizas iun per io.
```suggestion
        "Login.MissingLogin": "Mankas identigilo",
        "Login.MissingLogin.Description": "Ĉu vi forgesis doni identigilojn?",
```

Jackalgirl replied:
Mi kredas ke "provizi" ne estas malĝuste (per PIV, ĉi tie: https://vortaro.net/#provizi_kdc), sed mi tre preferas "mankas", dankon!

behilam commented on eo.json:
```diff
@@ -305,24 +371,135 @@
+        "Login.MissingLogin.Description": "Ĉu vi forgesis provizi itentigilojn?",
+        
+        "Login.InvalidRecoveryCode": "Nevalida restaŭrkodo",
+        "Login.InvalidRecoveryCode.Description": "Bonvolu kontroli vian resaŭrkodon kaj reprovi.",
```
```suggestion
        "Login.InvalidRecoveryCode.Description": "Bonvolu kontroli vian restaŭrkodon kaj reprovi.",
```

behilam commented on eo.json:
```diff
@@ -305,24 +371,135 @@
+        "Login.InvalidRecoveryCode": "Nevalida restaŭrkodo",
+        "Login.InvalidRecoveryCode.Description": "Bonvolu kontroli vian resaŭrkodon kaj reprovi.",
+        
+        "Login.UnsupportedAuthentication": "Nesubtenita metodo de aŭtentigo",
+        "Login.UnsupportedAuthentication.Description": "Ni ne subtenas ĉi tion metodon de aŭtentigo. Bonvolu provi alian.",
+
+        "Login.EmailNotVerified": "Retpoŝta addreso ne aprobita",
+        "Login.EmailNotVerified.Description": "Via retpoŝta adreso ne estas aprobita, bonvolu kontroli vian enirkeston por e-poŝton de ni, certe kontroli vian trudmesaĵ-dosierujon.",
+
+        "Login.AccountTempLocked": "Via konto estas ŝlosata.",
+        "Login.AccountTempLocked.Description": "Bonvolu kontakti Kontroladon je: moderation.resonite.com.",
+
+        "Login.AccountBanned": "Via konto estas barata.",
+        "Login.AccountBanned.Description": "Bonvolu kontakti Kontroladon je: moderation.resonite.com.",
```
```suggestion
        "Login.UnsupportedAuthentication": "Nesubtenita aŭtentig-metodo",
        "Login.UnsupportedAuthentication.Description": "Ni ne subtenas ĉi tiun aŭtentig-metodon. Bonvolu provi alian.",

        "Login.EmailNotVerified": "Retadreso ne aprobita",
        "Login.EmailNotVerified.Description": "Via retadreso ne estas aprobita. Bonvolu kontroli vian enirkeston por retmesaĝo nia. Ne forgesu kontroli vian spamujon.",

        "Login.AccountTempLocked": "Via konto estas ŝlosita.",
        "Login.AccountTempLocked.Description": "Bonvolu kontakti Moderigadon ĉe: moderation.resonite.com.",

        "Login.AccountBanned": "Via konto estas forbarita.",
        "Login.AccountBanned.Description": "Bonvolu kontakti Moderigadon ĉe: moderation.resonite.com.",
```

behilam reviewed (commented):
Bonega laboro! Mi kontrolis nur parton

Jackalgirl commented on eo.json:
```diff
@@ -27,22 +27,46 @@
+        "General.UI.Subtitles.On" : "Subtekstojn: Ŝalti",
+        "General.UI.Subtitles.Off" : "Subtekstojn: Malŝalti",
+
+        "General.UI.MusicVolumeLocal": "Musik-sonforteco (loka):",
+        "General.UI.VoiceVolumeLocal": "Voĉ-sonforteco (loka):",
+        "General.UI.SoundVolumeLocal": "Son-sonforteco (loka):",
```
Bonege, multan dankon!   Mi akordas kaj ĝisdatigos la dosieron, kaj ankaŭ aldonos vian nomon aŭ la aŭtorlisto. : )

Jackalgirl commented on eo.json:
```diff
@@ -52,22 +76,25 @@
         "Interaction.ScalingEnabled": "Skalado Ebligita",
         "Interaction.ScalingDisabled": "Skalado Malebligita",
-        "Interaction.ResetScale": "Restarigi skalon",
+        "Interaction.ResetScale": "Reŝarĝi Skalon",
```
Por "reset", ni uzas "restartigi" (per PIV/Komputeko, ĉi tie:  https://komputeko.net/#reset

Mi kredas ke "normala" eble estus subjecktiva termino; se vi skribas pri la restaŭro de la skalo de io -- hm, eble "restaŭri skalon"?

behilam replied:
Mi ŝatas! Estu do: `Restaŭri Skalon`

Jackalgirl commented on eo.json:
```diff
@@ -52,22 +76,25 @@
         "Interaction.Locomotion.None": "Neniu",
 
-        "Interaction.NamedEquipTool": "Ekuzi {name}",
+        "Interaction.NamedEquipTool": "Ekipi {name}",
```
Hm; en la dosiero mi vidas ke la vorto ja estas "ekipi"; ĉar ĉi tiuj frazoj aperos en kuntekst-menuojn, mi kredas ke estas grave, ke ili estu kiom mallonge tiom eble.  Mi opinias ke "ekipi" estas la plej bona vorto ĉi tie; oni ne nur prenos la ilon, sed ankaŭ ekipos ĝin -- mi kredas, estas diferenco inter "teni" aŭ "preni" kaj "ekipi".  Angle, "hold" or "take" estas malsamaj ol "equip".

behilam replied:
Jes, vi pravas. Plej bone ni uzu "ekipi", tamen ni ne povas diri "Ekipi skribilon" (ekzemple), ĉar tio signifus ke ni garnas la skribilon per io, ansataŭ tio ke ni ekipas nin per la skribilo.
Do per "ekipi" ni devas klarigi. Kion vi opinias pri `Ekipiĝi per {name}`?

Jackalgirl replied:
Ho, mi komprenas.  Mi nur vidis "(tr)" in PIV sed nun komprenas ke oni ekipas ricevanton per iuj ebleco.  Ni ekipas  mem per la ilo.  Bone, me ŝanĝos ĉie -- dankon!

Jackalgirl commented on eo.json:
```diff
@@ -81,34 +108,66 @@
+        "Inventory.OpenWorld": "Malfermi Mondon",
+        "Inventory.Equip": "Ekipi",
+        "Inventory.Delete": "Forviŝi",
+        "Inventory.CreateDirectory": "Kreii dosierujon",
```
Bona trovaĵo, dankon! : )

Jackalgirl commented on eo.json:
```diff
@@ -81,34 +108,66 @@
+        "Inventory.Share": "Kunhavigi",
+        "Inventory.Unshare": "Malkunhavigi",
+        "Inventory.SaveAvatar": "Konservi Avataron",
+        "Inventory.Favorite.Home": "Apliki Hejmon",
```
Mi akordas!  Estas pli bone ol "apliki" en ĉi tiu senco.  Mi ŝanĝos, dankon!

Jackalgirl commented on eo.json:
```diff
@@ -81,34 +108,66 @@
+        "Inventory.Favorite.ProgressBar": "Apliki Plenumskalon",
+        "Inventory.Favorite.AudioStreamController": "Apliki Flu-Regilon",
+        "Inventory.Favorite.TextDisplay": "Apliki Tekst-Ekranon",
+        "Inventory.Favorite.VideoPlayer": "Apliki Vid-Ludilon",
```
Bone, dankon!

Jackalgirl commented on eo.json:
```diff
@@ -81,34 +108,66 @@
+        "Inventory.Unshare": "Malkunhavigi",
+        "Inventory.SaveAvatar": "Konservi Avataron",
+        "Inventory.Favorite.Home": "Apliki Hejmon",
+        "Inventory.Favorite.ProgressBar": "Apliki Plenumskalon",
```
Mi akordas -- ŝanĝita, dankon denove!

Jackalgirl commented on eo.json:
```diff
@@ -81,34 +108,66 @@
+        "Inventory.Favorite.AudioPlayer": "Apliki Son-Ludilon",
+        "Inventory.Favorite.InteractiveCamera": "Apliki Fotilon",
+        "Inventory.Favorite.Keyboard": "Apliki Klavaron",
+        "Inventory.Favorite.Avatar": "Apliki Avataron",
+        "Inventory.Favorite.WorldLoadingIndicator": "Apliki Ŝarĝskalon",
```
ĉi tio ŝatas al mi -- multe pli bone.  Dankon!

Jackalgirl commented on eo.json:
```diff
@@ -81,34 +108,66 @@
         "Dash.Exit.ExitAndDiscard": "Eliri kaj Forĵeti Hejmojn",
+        "Dash.Exit.Sponsor": "Vastega dank' al {name} pro sponsori ĉi tiun projekton!",
+        "Dash.Exit.Sponsor.You": "VI",
+        "Dash.Exit.Shoutouts": "Ĉi tiu projekto ankaŭ eblas dank' al ĉi tiuj homoj:",
```
Bone, jes!

Jackalgirl commented on eo.json:
```diff
@@ -81,34 +108,66 @@
+        "Dash.Exit.Sponsor": "Vastega dank' al {name} pro sponsori ĉi tiun projekton!",
+        "Dash.Exit.Sponsor.You": "VI",
+        "Dash.Exit.Shoutouts": "Ĉi tiu projekto ankaŭ eblas dank' al ĉi tiuj homoj:",
+        "Dash.Exit.Shoutouts.AndMore" : " kaj pli multa!",
```
Mi kredis ke mi ja ŝanĝis ĉi tion, sed evidente ne!

Jackalgirl commented on eo.json:
```diff
@@ -135,11 +194,14 @@
+        "World.Error.NoPort": "Neniu Pordo Difinita",
+        "World.Error.IncompatibleVersion": "Nekongrua Versio",
+        "World.Error.HostBlocked": "Gastiganto barita",
+        "World.Error.HostBlockedDetail": "La gastiganton estas barita de vi",
+        "World.Error.WorldCrash": "La Mondo Kolapsis",
+        "World.Error.WorldCrashDetail" : "Neriparebla eraro dum la ĝisdatigo de la mondo",
+        "World.Error.Unknown": "Nekontata Eraro",
```
Bonege!  Ĝisdatigitajn. : )

Jackalgirl commented on eo.json:
```diff
@@ -235,10 +298,12 @@
         "World.AccessLevel.Anyone": "Iu ajn",
         "World.AccessLevel.RegisteredUsers": "Registritaj Uzantoj",
-
+        "World.AccessLevel.Contacts": "Kuntaktoj",
+        "World.AccessLevel.ContactsPlus": "Kuntaktoj+",
```
Mi luktis per la vorto "Kuntaktoj" kaj "Kuntaktuloj", sed nun ke vi opinias pri la vorto, mi akordas ke "Kuntaktuloj" estu pli bone.  Mi ŝanĝos ĉiudosiere. : )

Jackalgirl commented on eo.json:
```diff
@@ -305,24 +371,135 @@
-        "Indicator.ServerStatus": "<color=#fff>Servila stato:</color> {status, select, Good {Bona} Slow {Malrapida} Down {Malsupre} NoInternet {Neniu interreto}} ({response_ms} ms)",
+        "Indicator.OnlineUsers": "<color=hero.yellow>{users}</color> {users,plural, one {Enreta Uzanto} other {Enretaj Uzantoj}}",
+        "Indicator.OnlineUsersVR": "<color=hero.purple>{users}</color> {users, plural, one {uzanto} other {uzantoj}} en VR",
+        "Indicator.OnlineUsersDesktop": "<color=hero.cyan>{users}</color> {users, plural, one {uzanto} other {uzantoj}} en Labortabla",
```
Por ĉi tiu, mi kredas ke ni konservu la vorton "labortablo" (per https://komputeko.net/#desktop) -- personoj kiuj uzas Resoniton pre VR ankaŭ uzas komputilon; la diferenco ĉi tie estas ke ĉu la uzanto estas en VR-reĝimo aŭ "desktop" (angle).  Sed eble "en Fenestrila" pli bonus?  Kiel vi opinias?

behilam replied:
Angle "desktop" estas la vorto por paroli pri sur-tabla komputilo, kaj ankaŭ por la baza enkran-enhavo.
En Esperanto "labortablo" estas por la dua. Ĝi estas iom figura uzo ĉar tio estas la baza loko kie vi laboras pri la komputilo, via "labortablo"
| Labortablo | Sur-tabla komputilo |
|-|-|
| ![image](https://github.com/Yellow-Dog-Man/Locale/assets/73665457/5232ed7c-20d9-406d-a575-13d59345079d) | ![image](https://github.com/Yellow-Dog-Man/Locale/assets/73665457/77f91a6e-38b7-485c-b63c-0af5a78f5427) |

Do ne temas pri "labortablaj" uzantoj, sed pri "ĉe-komputilaj" uzantoj, ĉar ili estas ĉe komputilo, anstataŭ per VR

"Fenestrila" iom ambiguas, ĉar plenekrane por kelkaj ne plu temas pri fenestro. Mi preferus "ĉe-komputilaj". Ĉu vi ŝatas?

Jackalgirl commented on eo.json:
```diff
@@ -305,24 +371,135 @@
+        "Indicator.OnlineUsersVR": "<color=hero.purple>{users}</color> {users, plural, one {uzanto} other {uzantoj}} en VR",
+        "Indicator.OnlineUsersDesktop": "<color=hero.cyan>{users}</color> {users, plural, one {uzanto} other {uzantoj}} en Labortabla",
+
+        "Indicator.ServerStatus.Status": "Servila stato",
```
Bone!

Jackalgirl commented on eo.json:
```diff
@@ -305,24 +371,135 @@
+        "Indicator.Sync.StorageFull": "Konservejo plenas! <size=61.803%>Ne povas sinkronigi</size>",
+
+        "Indicator.Storage.StorageFull": "Konservejo Plenas",
+        "Indicator.Storage.Usage": "{storage_used} de {storage_max}<br><size=61.803%>de Space en la Konservejo Uzita</size>",
```
Ankaŭ bone, dankon!

Jackalgirl commented:
Just finished a final line-by-line and loaded this into Resonite with no issues.  Will go ahead and "undraft" it so we can get moving. : )

Frooxius commented:
Unfortunately this has conflicts with the main branch. I'll need to you to resolve these on your end, I don't know the language, so I'm unable to resolve them myself.

Frooxius commented:
Ok looks good, merging! Thank you!
